tests/testthat/test-condense_control.R

```diff
@@ -0,0 +1,52 @@
+test_that("control functions respect hierarchy with condense_control", {
+
+  ctrl <- parsnip::condense_control(
```

Comment:
I'm assume here that `control_grid()` is in theory a super of `control_resamples()`, they just happen to be equal at this point in time